### DISCUSSION TITLE
can we have two dictionaries

### DISCUSSION THREAD

**Discussion by ChristianET-DS** [wolfgarbe/symspell#117] [Q&A]
Hi 
for one german text , which contains english words like Corporate or Solution.

I would like to have two dictionaries. 
Otherwise solution is union in german,

is it possible to have this two dictionaries together ?
It can work?

**Comment by wolfgarbe** [comment]
You can combine a German and an English dictionary into a single dictionary (with mixed German and English vocabulary). 

There is a slight problem with this solution. When the vocabulary is doubled, then the number of suggestions increases, and the precision of suggestions decreases - especially if German and English words are close:

feld [DE], field [EN]
horn [DE], korn [DE], corn [EN]

For the misspelled term "orn" SymSpell doesn't know whether it should correct it to korn [DE] or corn [EN], so it will display both suggestions.
The term "feld" could be a correct German word or a misspelled English "field" ...
